src/jit/compiler.cpp

```diff
@@ -7202,7 +7225,8 @@ void JitTimer::PrintCsvHeader()
 
     CritSecHolder csvLock(s_csvLock);
 
-    if (_waccess(jitTimeLogCsv, 0) == -1)
+    FILE* fp = _wfopen(jitTimeLogCsv, W("r"));
+    if (fp == nullptr)
```

Comment:
I am now also confused - I've no idea why something would appear twice (if I understand your comment correctly). Maybe I should just squash everything when I commit / push the latest changes, since there seems to be no benefit in preserving the little pieces.